lib/kernel/host/CMakeLists.txt

```diff
@@ -342,6 +342,10 @@ else()
   set(KERNEL_SOURCES ${SOURCES_GENERIC})
 endif()
 
+if(HOST_CPU_ENABLE_CL_KHR_FP16)
```

Comment:
There is already an existing if condition on line 245, please use that one. Also, add the removal of libclc isinf in the if block.

Reply:
Relocated. libclc's isinf is preserved and the `isinf_half.cl` complements it by adding implementation for the half overload. Agreed it's non-ideal, it's just for quick fix until we decide how we go forward with the kernel builtins.